integration-tests/integration/createIncident/incident-creation.spec.js

```diff
@@ -16,13 +17,15 @@ context('Logging in', () => {
     const tasklistPage = TasklistPage.visit(bookingId)
     tasklistPage.checkNoPartsComplete()
     const newIncidentPage = tasklistPage.startNewForm()
+    newIncidentPage.fillForm()
     const detailsPage = newIncidentPage.save()
     detailsPage.fillForm()
     const relocationAndInjuriesPage = detailsPage.save()
     relocationAndInjuriesPage.fillForm()
     const evidencePage = relocationAndInjuriesPage.save()
     evidencePage.fillForm()
-    evidencePage.save()
+    const checkInputs = evidencePage.save()
```

Comment:
maybe call this 'checkAnswersPage' - this is what we refer to it in other places

Reply:
I had called it checkInputs rather than checkAnswersPage because checkAnswersPage is used as a const a few lines below. 
So, Have done as above and removed the const from line 33 so that checkAnswersPage can be reassigned thus: checkAnswersPage = tasklistPageAfterAllPartsComplete.goToAnswerPage()